### ISSUE TITLE
Sync warning message formatting - follow up to 4132

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#4490]
## Description 
Follow up to https://github.com/brave/brave-browser/issues/4132

Issue above states that both code words and QR code screens should display message that the sync key is sensitive with some degree of bolding to the message (see description and https://github.com/brave/brave-browser/issues/4132#issuecomment-484234764). Message currently blends in to the rest of the text on the modal.

## Steps to Reproduce

   1. Navigate to brave://sync
   2. Select Start a new Sync Chain
   3. Select 'Computer', you will see added message as dictated in https://github.com/brave/brave-browser/issues/4132#issuecomment-484234764 except for bolding. QR code screen should display this message as well, but currently does not.

## Actual result:
![Screen Shot 2019-05-17 at 4 39 57 PM](https://user-images.githubusercontent.com/28145373/57956394-cd9d3300-78c6-11e9-9776-c4fd96cc2d41.png)

## Expected result:
Expecting bold text per https://github.com/brave/brave-browser/issues/4132#issuecomment-484234764. If this is no longer the expected result, please close this issue.

## Reproduces how often: 
easily

## Brave version (brave://version info)

Brave | 0.65.95 Chromium: 74.0.3729.131 (Official Build) beta(64-bit)
-- | --
Revision | 518a41c1fa7ce1c8bb5e22346e82e42b4d76a96f-refs/branch-heads/3729@{#954}
OS | Mac OS X

## Version/Channel Information:

- Can you reproduce this issue with the current release? n/a
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the dev channel? yes
- Can you reproduce this issue with the nightly channel? yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? n/a
- Does the issue resolve itself when disabling Brave Rewards? n/a
- Is the issue reproducible on the latest version of Chrome? n/a

**Comment by LaurenWags** [comment]
cc @diracdeltas as original issue had `security` tag.

**Comment by rebron** [comment]
Good catch @LaurenWags We should have the "Treat this code like a password" in bold.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
